src/components/TooltipController/TooltipController.tsx

```diff
@@ -235,6 +237,22 @@ const TooltipController = ({
     }
   }, [anchorRefs, providerActiveAnchor, activeAnchor, anchorId, anchorSelect])
 
+  useEffect(() => {
+    if (process.env.NODE_ENV === 'production') {
+      return
+    }
+    if (style?.border) {
+      // eslint-disable-next-line no-console
+      console.warn('[react-tooltip] Do not set `style.border`. Use `border` prop instead.')
+    }
+    if (border && !cssAttrIsValid('border', border)) {
+      // eslint-disable-next-line no-console
+      console.warn(
+        `[react-tooltip] "${border}" is not a valid \`border\`. See https://developer.mozilla.org/en-US/docs/Web/CSS/border`,
+      )
+    }
+  }, [])
```

Comment:
I have the feeling that this looks a bit verbose

Reply:
We can drop the MDN docs link then.